analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/FilesToAnalyze/InvalidValuesFileToAnalyze.txt

```diff
@@ -0,0 +1,4 @@
+﻿foo
```

Comment:
This file should be renamed to `InvalidValuesFilesToAnalyze.txt` (missing `s`, it was hard to find with filter in Solution explorer)

Reply:
I renamed the file to FilesToAnalyzeWithInvalidValues